packages/components/input/__tests__/input.spec.tsx

```diff
@@ -458,9 +458,33 @@ describe('Input.vue', () => {
     const input = wrapper.find('input')
     const textarea = wrapper.find('textarea')
     await nextTick()
+    await nextTick()
```

Comment:
Is this a special hack? @JeremyWuuuuu

Reply:
> Is this a special hack? @JeremyWuuuuu

Maybe `nextTick` is used in my code, so that this unit test must add `await nextTick()`  to pass. Now I have modified the code, and this unit test has been modified back to the previous version